doc/neps/roadmap.rst

```diff
@@ -17,6 +17,12 @@ facilitate interoperability with all such packages, and the code that uses them,
 may include (among other things) interoperability protocols, better duck typing
 support and ndarray subclass handling.
 
+The key goal is: *make it easy for code written for NumPy to also work with
+other NumPy-like projects.* This will enable GPU support via, e.g, CuPy or JAX,
+distributed array support via Dask, and writing special-purpose arrays (either
+from scratch, or as a ``numpy.ndarray`` subclass) that work well with SciPy,
+scikit-learn and other such packages.
+
```

Comment:
This seems like a restatement of the previous paragraph. Do you intend that or would it make sense to merge them?

Reply:
I meant to keep it separate. That first paragraph made sense to insiders, but I'm pretty sure it was way too difficult for an average interested user to get from that what the second paragraph now says. So different audiences.

If we can merge them, great - but it's not too easy to do I found.